integration_tests/src/tests/replica_update_test.rs

```diff
@@ -149,11 +151,6 @@ pub fn upgrade_and_miss_post_test(v: &dyn VM) {
     assert_invariants(v, &Policy::default(), None)
 }
 
-#[vm_test]
-pub fn upgrade_and_miss_post(v: &dyn VM) {
-    upgrade_and_miss_post_test(v);
```

Comment:
keep?

Reply:
This just calls the function `upgrade_and_miss_post_test`, line 102 above. I added the annotation there. The call site was already calling it there.

(This code is a remnant from when there was another parameter to the test method, but vm_test doesn't support it)

Reply:
Ah, got it.